eng/Directory.Build.Common.props

```diff
@@ -116,7 +116,7 @@
     <SupportsNetStandard20 Condition="'$(SupportsNetStandard20)' == ''">false</SupportsNetStandard20>
 
     <RequiredTargetFrameworks>net452;netstandard1.4;net461;netstandard2.0</RequiredTargetFrameworks>
-    <RequiredTargetFrameworks Condition="'$(SupportsNetStandard20)' == 'true'">netstandard2.0</RequiredTargetFrameworks>
+    <RequiredTargetFrameworks Condition="'$(SupportsNetStandard20)' == 'true'">netstandard2.0;net6.0</RequiredTargetFrameworks>
```

Comment:
This file needs to use ``$(MSBuildProjectName)`` to log when a project is missing one of these required tfm's so that way it's easier to identify *which one* is failing the build.